applications/byte_patrol/middleware.py

```
@@ -140,8 +172,26 @@
         """
         Process each request to check if IP restriction should be applied.
         """
+        # Add detailed request logging
+        logger.debug(f"BytePatrol: Processing request for path: '{request.path}'")
+        logger.debug(f"BytePatrol: Request method: {request.method}")
+        logger.debug(
+            f"BytePatrol: Request path info: {request.META.get('PATH_INFO', 'N/A')}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)